### DISCUSSION TITLE
Metadb has issues after upgrading from 1.10.10 to 2.0.2

### DISCUSSION THREAD

**Discussion by HashSalmon** [apache/airflow#16146] [Q&A] [answered]
Hey, I upgraded our airflow from 1.10.10 to 2.0.2 by following the instruction provided here https://airflow.apache.org/docs/apache-airflow/stable/upgrading-to-2.html. However, after running `airflow db upgrade`. It seems like the db migration did not upgrade peacefully. The problem I have at this moment is: let's say I have 50 DAGs but only 45 of 50 DAGs works fine. You can click the DAG from the UI. The rest of 5 DAGs, if you click them on the UI page. There will be an error page showing up with message

`raise SerializedDagNotFound(f"DAG '{dag_id}' not found in serialized_dag table")
airflow.exceptions.SerializedDagNotFound: DAG 'dynamic_dag_1' not found in serialized_dag table`

I tried to remove the 5 broken DAGs from UI but the airflow did not load them from dags folder. 

The scheduler has some error messages

![Screen Shot 2021-05-28 at 9 34 46 AM](https://user-images.githubusercontent.com/2868104/120015293-05501280-bf98-11eb-96cb-eb4031e290e1.png)

I am not sure how to solve this issue. I don't want to rebuild the metadb. Is there a way to fix the metadb? Thanks!

**Comment by potiuk** [comment, marked as answer]
It looks like your dags contain strange characters somewhere in description (\uff0c -> https://www.fileformat.info/info/unicode/char/ff0c/index.htm). However your database uses `latin1` encoding and is not able to store them. You should either find and fix the DAGs (replace those special characters and any other special characters with regular ",") or change the encoding of your database to "utf8_mb3" - this is safest bet because it will be able to store nearly any character.

**Reply by HashSalmon** [reply]
Hey, thanks for your help. Let me give it a try and will update this ticket later.

**Reply by HashSalmon** [reply]
Hey, thanks for the hints. I removed the full width comma from the dag's py doc section and it works.
